tests/app/main/test_services.py

```diff
@@ -1216,7 +1216,7 @@ def test_delete_button_redirects_with_are_you_sure(self, data_api_client):
         assert_in('/frameworks/g-cloud-7/submissions/scs/1?delete_requested=True', res.location)
         res2 = self.client.get('/suppliers/frameworks/g-cloud-7/submissions/scs/1?delete_requested=True')
         assert_in(
-            b"Are you sure you want to delete this service?", res2.get_data()
+            b"Are you sure you want to delete this lab?", res2.get_data()
```

Comment:
:cry: 

This is a test for an SCS service that is now expecting a wrong thing.